boards/arduino-mega2560/include/arduino_pinmap.h

```diff
@@ -16,7 +16,7 @@
  * You can use the defines in this file for simplified interaction with the
  * Arduino specific pin numbers.
  *
- * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Daniel Nordahl <nordahl.d@gmail.com>
```

Comment:
We usually just add new authors, unless you basically replace the whole file. ;)